### PR TITLE
fix: resolve incorrect slack messages

### PR DESCRIPTION
**Pull request by james-a-morris**
This PR adds an upgrade to the SDK where we are now correctly generating slack markdown

### CODE REVIEW

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by pxrl — APPROVED** [review]
Also LGTM! Nice to get this fixed.

File: scripts/constructEmergencyRoot.ts [hunk 1]
```diff
@@ -68,19 +67,19 @@ export async function run(): Promise<void> {
   const poolRebalanceLeaves: PoolRebalanceLeaf[] = [
     {
       chainId: 1,
-      bundleLpFees: [toBN("124122273836729709").sub(toBN("123603942018663986"))],
-      netSendAmounts: [toBN("120200463420953608591").sub(toBN("119648808743556644079"))],
-      runningBalances: [toBN(0)],
+      bundleLpFees: [sdkUtils.toBN("124122273836729709").sub(sdkUtils.toBN("123603942018663986"))],
```
[the remark this answers is not in the document]
Comment (james-a-morris) [reply scripts/constructEmergencyRoot.ts:70]: Fair enough - I can rework this

**Review by pxrl — APPROVED** [review]
LGTM! There might be some hangover in TransactionUtils.ts but it's basically only cosmetic / minor style.

**Review by pxrl — APPROVED** [review]
++LGTM.

**Pull request merged**